src/serac/physics/solid_mechanics_contact.hpp

```diff
@@ -240,14 +249,26 @@ class SolidMechanicsContact<order, dim, Parameters<parameter_space...>,
     nonlin_solver_->solve(augmented_solution);
     displacement_.Set(1.0, mfem::Vector(augmented_solution, 0, displacement_.Size()));
     contact_.setPressures(mfem::Vector(augmented_solution, displacement_.Size(), contact_.numPressureDofs()));
+    contact_.update(cycle_, ode_time_point_, dt);
+    forces_.SetVector(contact_.forces(), 0);
+  }
+
+  void warmStartDisplacement()
+  {
+    SolidMechanicsBase::warmStartDisplacement();
+    auto dt = 1.0;
+    contact_.update(cycle_, time_, dt);
```

Comment:
I think the contact warmStart needs to include the contact forces and stiffnessess.  But this can wait for another day.

Reply:
These commits were cherry-picked from a much messier working tree and this apparently slipped through